components/layout/DocsLayout.js

```diff
@@ -107,8 +108,11 @@ export default function DocsLayout({ post, navItems = {}, children }) {
               </div>
             </div>
           </main>
+          <Footer/>
```

Comment:
@ritik307 Hello! Nice try, but please look at example docs page like here:

![image](https://user-images.githubusercontent.com/20404945/151795442-65adf3cc-72ab-4fc5-9529-5c1f6026e8a9.png)

Footer shouldn't broke other parts of page, it should be localize on the bottom of page like in other pages like in `tools/*`.